x-pack/plugins/ml/public/application/model_management/test_models/utils.ts

```diff
@@ -31,5 +39,9 @@ export function isTestable(modelItem: ModelItem, checkForState = false) {
     return true;
   }
 
+  if (isDfaTrainedModel(modelItem)) {
+    return true;
+  }
+
   return false;
```

Comment:
This could be just:

```ts
return isDfaTrainedModel(modelItem);
```

Reply:
Updated in 2c51898d8c7022455d87e5815e92eb7880e092f9